daemon/main.rs

```diff
@@ -178,6 +179,7 @@ mod tests {
             cache_root: temp.join("cache").to_string_lossy().into(),
             state_root: temp.join("state").to_string_lossy().into(),
             wallet_address: "f1test".to_string(),
+            station_id: "zinnia-dev".to_string(),
```

Comment:
I think that only passing through the environment variable "station_id" isn't ideal, as Zinnia ideally doesn't know anything about Station. Instead there should be a way to pass any key value pair into the module context. I believe that implementing this is out of scope for this PR, so just noting here that I would like to improve this in the future.